lib/ansible/modules/cloud/docker/_docker.py

```diff
@@ -25,7 +25,7 @@
 description:
   - This is the original Ansible module for managing the Docker container life cycle.
   - NOTE - Additional and newer modules are available. For the latest on orchestrating containers with Ansible
-    visit our Getting Started with Docker Guide at U(https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/guide_docker.rst).
+    visit our L(Getting Started with Docker Guide,.../scenario_guides/guide_docker.html).
```

Comment:
I think there's a dot too many:
```suggestion
    visit our L(Getting Started with Docker Guide,../scenario_guides/guide_docker.html).
```

Reply:
I just built the webdocs locally and tested it. two dots is right. Thnx @felixfontein